tests/topKgate_old.py

```diff
@@ -0,0 +1,279 @@
+# -------------------------------------------------------------------------
```

Comment:
this file again seems to be an older version of topKgate. May we remove it if not needed?

Reply:
Thanks for careful review. Removed now

Reply:
Same. I added it back. Current test actually depend on this file.